community/contributing/bisecting_regressions.rst

```diff
@@ -105,16 +147,23 @@ Compile Godot. This assumes you've set up a build environment:
 
 .. code-block:: shell
 
-    # <platform> is the platform you're targeting for regression testing,
-    # like "windows", "x11" or "macos".
-    $ scons platform=<platform> -j4
+    $ scons
 
 Since building Godot takes a while, you want to dedicate as many CPU threads as
 possible to the task. This is what the ``-j`` parameter does. Here, the command
 assigns 4 CPU threads to compiling Godot.
```

Comment:
Removing the `-j4` parameter in the code block, makes this describing text inconsistent, which refers to the `-j` parameter.

Reply:
Fixed. Godot's SCons setup now detects how many CPU threads should be used depending on the system, but it intentionally leaves 1 thread available on systems with at least 4 threads to make multitasking smoother. There's a fair chance people will be doing something else while waiting for Godot to compile (speaking from experience), so I think this is a good default for bisecting regressions.